tests/integration/devfile/cmd_devfile_init_test.go

```diff
@@ -62,6 +62,7 @@ var _ = Describe("odo devfile init command tests", func() {
 	Context("running odo init with valid flags", func() {
 		When("using --devfile flag", func() {
 			BeforeEach(func() {
+				helper.CreateTelemetryDebugFile()
```

Comment:
Why do you add it here, when you do not test the resulting file content?

Reply:
Must be leftover from rebase, removing